crosswalks/Composer.csv

```diff
@@ -0,0 +1,68 @@
+Property,Packagist
+codeRepository,support.source
+programmingLanguage,
+runtimePlatform,
+targetProduct,
+applicationCategory,type
+applicationSubCategory,
+downloadUrl,
+fileSize,
+installUrl,
+memoryRequirements,
+operatingSystem,
+permissions,
+processorRequirements,
+releaseNotes,
+softwareHelp,
+softwareRequirements,

```

Comment:
https://getcomposer.org/doc/04-schema.md#require

Reply:
Updated it. Thank you!